edx_proctoring/tests/utils.py

```diff
@@ -307,7 +311,7 @@ def _create_exam_attempt(self, exam_id, status=ProctoredExamStudentAttemptStatus
         if ProctoredExamStudentAttemptStatus.is_completed_status(status):
             attempt.completed_at = datetime.now(pytz.UTC)
 
-        if status == ProctoredExamStudentAttemptStatus.error:
+        if status == ProctoredExamStudentAttemptStatus.error and not (ready_to_resume or resumed):
```

Comment:
I think you can use the `is_attempt_in_resume_process` function to keep that logic in one place